pages/agent/v3/cli_secret.md

```diff
@@ -0,0 +1,9 @@
+# `buildkite-agent secret`
+
+> Note: The `secrets` command, as well as its associated server-side features, are currently in feature preview. If you're interested in trialing Buildkite Secrets, [contact support](mailto:support@buildkite.com).
```

Comment:
Thanks very much for your contribution @moskyb !

For consistency with the rest of the Buildkite Docs, I'd recommend sticking to our existing [style guide for callouts/admonitions](https://github.com/buildkite/docs/blob/main/styleguides/markdown-syntax-style.md#callouts):

```suggestion
> 📘 The `secrets` command, as well as its associated server-side features, are currently in feature preview.
> If you're interested in trialing Buildkite Secrets, [contact support](mailto:support@buildkite.com).
```